jvm-libs/generic/json-rpc/src/main/kotlin/net/consensys/linea/jsonrpc/JsonRpcMessageProcessor.kt

```diff
@@ -64,17 +60,10 @@ class JsonRpcMessageProcessor(
     DatabindCodec.mapper().registerKotlinModule()
   }
 
-  private val counterBuilder = Counter.builder("jsonrpc.counter")
   override fun invoke(user: User?, messageJsonStr: String): Future<String> =
     handleMessage(user, messageJsonStr)
 
   private fun handleMessage(user: User?, requestJsonStr: String): Future<String> {
-    val wholeRequestTimer =
-      Timer.builder("jsonrpc.processing.whole")
-        .description(
-          "Processing of JSON-RPC message: Deserialization + Business Logic + Serialization"
-        )
-    val timerSample = Timer.start(Clock.SYSTEM)
```

Comment:
The time sample was taking the time here right at method beginning. Now it's move elsewhere. That will influence time measurements, no?

If read the code correctly, for instance now the time doing `decodeMessage(requestJsonStr)`

Reply:
Yes, I revised it a bit and the `jsonrpc.processing.whole` metric should now also capture the time spent for decoding and parsing of the request messages.

Reply:
We need to be very careful with kind of changes/regressions. Metrics are very important, we take improment/performance and troubleshooting  decisions based on them. If the values are not accurate it will lead to wrong decisions and time waste.